server/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -836,7 +836,7 @@ public AnalyzedRelation visitValues(Values values, StatementAnalysisContext cont
 
         var normalizer = EvaluatingNormalizer.functionOnlyNormalizer(
             nodeCtx,
-            f -> f.signature().isDeterministic()
+            Symbols::isDeterministic
```

Comment:
Maybe nitpicking:

To my understanding we first replaced all `isDeterministic` usages in the first commit to check also arguments. |

In the second commit we stopped passing `function-> true` and normalizing depending on function being deterministic or not. 

If so, this change probably belongs to the first commit.

Reply:
Thanks for checking, I just had a second look. The first commit is actually wrong.

Reply:
as discussed we can remove first commit and this change. Thanks.